### DISCUSSION TITLE
FATES Simulation in Desert Environments

### DISCUSSION THREAD

**Discussion by LianGong98** [NGEET/fates#1258] [Q&A] [answered]
Hello all:
  I am interested in using FATES in extreme arid desert environments, and a critical issue is how to ensure plant survival under such harsh conditions. I have noticed that in ELM-FATES, the soil profile consists of only 10 layers, totaling 3.5 meters in depth. When I attempted to simulate using the static mode (use_fates_ed_st3), all plants died due to insufficient water availability.

  Does FATES takes into account the capability of plants to access groundwater in extreme arid regions? Additionally, do you have any recommendations on how to enhance the model to allow plants to effectively access adequate subsurface water?

Thank you for your assistance!

Best regards!

**Comment by rosiealice** [comment, marked as answer]
Hi @LianGong98 thanks for this question. In general, FATES defers to the host land model in terms of the soil profile. In CLM/CTSM the soil profile is variable depth and can be a lot deeper than 3.5m but maybe this capability isnt included in ELM thus far. I guess people running ELM-FATES over the amazon (I forget who is doing that in NGEET) might have encountered similar needs to access deep water (@ckoven I forget who is doing this). 

A mor philosophical question for semi arid regions that I have thought about a lot but not worked on (as it is hard) is whether vegetation can survive at all on the mean soil moisture in a semi arid sytem. In real life, vegetation is spatially patchy in such places and there are lots of means (via roots, and topography) to enable moisture convergence and vegetation survival.   This is quite a hard thing to imagine representing in a land surface model, as it would require some kind of bespoke moisture tiling scheme (one that works differently to the hydrological tiling that now exists in CLM or the topopgraphical tiling in ELM) but it is probably quite important for getting these systems right...

Anyway, that thought is mission creep on your original question but we have not run FATES in detail in semi arid systems so I just wanted to flag it.. All LSMs would in theory suffer from this problem, not just FATES.
